supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc

```diff
@@ -189,11 +189,13 @@
 *See also*:
 
 [[now]]
-==== image:images/yes.png[yes] now (adverb)
-*Description*: "Now" means at the present time, immediately, or at once.
+==== image:images/caution.png[caution] Now (adverb)
+*Description*: In most content, you can use "now" when you mean the present time, immediately, or at once.
 
-*Use it*: yes
+However, in release notes, avoid using "now" to refer to the current release. Instead, use "With this update," as shown in this guide's xref:release-notes[Release notes] section. This is because, in release notes that might span multiple releases, the temporal context of "now" can become ambiguous and cause confusion.
```

Comment:
```suggestion
However, in release notes, avoid using "now" to refer to the current release. Instead, use "with this update," as shown in this guide's xref:release-notes[Release notes] section. This is because, in release notes that might span multiple releases, the temporal context of "now" can become ambiguous and cause confusion.
```

Reply:
Updated. Thanks.